pil-analyzer/src/pil_analyzer.rs

```diff
@@ -125,7 +126,7 @@ impl PILAnalyzer {
         }
     }
 
-    pub fn process(&mut self, mut files: Vec<PILFile>) {
+    pub fn process(&mut self, mut files: Vec<PILFile>) -> Result<(), Vec<Error>> {
```

Comment:
There are still a lot of panics in functions called from here. Created issue: https://github.com/powdr-labs/powdr/issues/1757